### PR TITLE
Table component

### PR DESCRIPTION
**Pull request by mariush2**
- [AB#431636](https://dev.azure.com/Equinor/4319082b-29ba-4111-a20b-c04f55c7e7c6/_workitems/edit/431636)

### CODE REVIEW

File: src/components/DataDisplay/Table/Table.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    `
+    cursor: default;
+    tr:hover, th:hover {
+      background: unset;
+      cursor: default;
+    }
+    `}
+`;
+
+function Table<T>(props: EdsDataGridProps<T>) {
```
[the remark this answers is not in the document]
Comment (mariush2) [reply src/components/DataDisplay/Table/Table.tsx:59]: The generic type thingy isn't really possible (AFAIK) when doing `const Name => () { ... }`

[the remark this answers is not in the document]
Comment (aslakihle) [reply src/components/DataDisplay/Table/Table.tsx:59]: This is on OptionDrawer:
image

[the remark this answers is not in the document]
Comment (mariush2) [reply src/components/DataDisplay/Table/Table.tsx:59]: image
Like this?

[the remark this answers is not in the document]
Comment (mariush2) [reply src/components/DataDisplay/Table/Table.tsx:59]: Looks kinda strange with the `<T,>` (doesn't let me just have `<T>`)

[the remark this answers is not in the document]
Comment (aslakihle) [reply src/components/DataDisplay/Table/Table.tsx:59]: Weird 🤔 
But yes, this is what i was imagining, just with FC aswell, but maybe not needed on a component like this 🤔

**Review by aslakihle — APPROVED** [review]

**Pull request merged**